bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClient.java

```diff
@@ -290,11 +290,12 @@ public Request createRequest(String url, HttpMethod method, @Nullable Object con
             if (errorResponseHandler != null) {
                 throw errorResponseHandler.apply(statusCode, textContent);
             } else {
-                throw new ExecutionException(String.format("Request failed with status code %s", statusCode), null);
+                throw new ExecutionException(String.format("Send request failed with status code %s", statusCode),
+                        null);
```

Comment:
Should we start to localize exception messages? They are often forwarded to the UI as they are, so it might be a good idea if they are localized.

Reply:
I suggest to do this within an own issue.